util/filter_bench.cc

```diff
@@ -491,8 +529,7 @@ double FilterBench::RandomQueryTest(uint32_t inside_threshold, bool dry_run,
 
   uint32_t num_infos = static_cast<uint32_t>(infos_.size());
   uint32_t dry_run_hash = 0;
-  uint64_t max_queries =
-      static_cast<uint64_t>(FLAGS_m_queries * 1000000 + 0.50);
+  uint64_t max_queries = static_cast<uint64_t>(m_queries_ * 1000000 + 0.50);
```

Comment:
It is possible that `m_queries_ * 1000000` overflows. Cast m_queries_ to uint64_t first.

Reply:
The real bug is accidentally declaring the new m_queries_ member as uint32_t instead of double like the command line parameter. (Sometimes I like to do fractions of millions of queries, but usually millions.)

And for whatever reason, neither our gcc or clang compilers complain about implicit narrowing conversion on `m_queries_ = FLAGS_m_queries;` 😠